pkg/cluster/spec.go

```diff
@@ -179,6 +180,11 @@ func (s *Spec) RootVersionsBundle() *VersionsBundle {
 	return s.VersionsBundle(s.Cluster.Spec.KubernetesVersion)
 }
 
+// FirstVersionsBundle returns the VersionsBundle for the Cluster specs first Bundles.Spec.VersionBundles.
+func (s *Spec) FirstVersionsBundle() *releasev1.VersionsBundle {
```

Comment:
This function looks redundant, it may be easier to just "Bundles.Spec.VersionsBundles[0]"

Reply:
I did this at first because I found it easier to read about, but now I'm having second thoughts.

Reply:
Updated